joint_limits:
  drive_wheel_joint:
    has_position_limits: false
    has_velocity_limits: true
    max_velocity: 4.2
    has_acceleration_limits: false
    max_acceleration: 10.0
    has_jerk_limits: false
    max_jerk: 100.0

  steer_joint:
    has_position_limits: false
    has_velocity_limits: true
    max_velocity: 4.2
    has_acceleration_limits: false
    max_acceleration: 10.0
    has_jerk_limits: false
    max_jerk: 100.0

  fork_joint:
    has_position_limits: false
    has_velocity_limits: true
    max_velocity: 0.8
    has_acceleration_limits: false
    max_acceleration: 10.0
    has_jerk_limits: false
    max_jerk: 100.0
